chore: Document default strategy (#4010)
Unleash/unleash: website/docs/reference/project-default-strategy.md

Author: sure makes sense

---
title: Project default strategy
---

> 

Reviewer: I suggest using figures for some of the images. To do that, we need to import the component.

```suggestion
import Figure from '@site/src/components/Figure/Figure.tsx'

```

:::info Availability

The project default strategy feature is generally available starting with **Unleash 5.2.0**.

:::

A project's environment's **default strategy** specifies **the activation strategy that will be used when you enable a feature in that environment, if the environment has no defined strategies)**. 

> ## Limitations
> 
> The default project strategy uses the [Gradual Rollout activation strategy](/reference/activation-strategies.md) as it's base.  You can customise to your needs by using constraints or segments (enterprise only)

Reviewer: This header doesn't seem to apply to the text, so I think we can just remove it. (Plus some minor text suggestions and links)

```suggestion
All default project strategies use the [gradual rollout activation strategy](activation-strategies.md). By default, the rollout 100%. You can customize the strategies by changing the rollout percentage and adding [constraints](strategy-constraints.mdx) and [segments](segments.mdx) as you would for any other strategy.
```

> ## Setting up

Reviewer: ```suggestion
## Configuration
```

> When in project overview, go to the Project settings tab and click on Default strategy 

Reviewer: ```suggestion
Custom strategies are configured from each project's project settings tab.
```

> ![project_settings_default_strategy.png](/img/project_settings_default_strategy.png)

Reviewer: ```suggestion
<Figure caption="The default strategy configuration page is available from the project settings tab." img="/img/project_settings_default_strategy.png" />
```

> You can customise the default strategy for each of the project's environments by clicking on the edit button
> 
> ![edit_default_Strategy.png](/img/edit_default_Strategy.png)

Reviewer: ```suggestion
The default strategies screen lists a strategy for each of the project's environments

<Figure caption="Each strategy can be individually configured with the corresponding edit button." img="/img/edit_default_Strategy.png" />
```

> 
> This will open up the familiar activation strategy form, where you can add your constraints and/or segments
> 
> ![activation_Strategy_form.png](/img/activation_Strategy_form.png)

Reviewer: I don't think we need this, but am happy to hear your thoughts otherwise.
```suggestion
```

Author: makes sense
